src/components/App/AppSidebar/index.vue

```diff
@@ -96,18 +96,31 @@ $transition: transform 700ms;
 }
 
 .AppSidebar-content {
-  @include flex-container(column, space-between, flex-start);
   height: 100%;
   padding: 40px 20px 40px 30px;
+  position: relative;
+
+  @media only screen and (min-width: 768px) and (max-width: 1200px) and (orientation: landscape) {
```

Comment:
can you set this (768px, 1200px) to vars that are set in config? like $mobileBreakpoint?